docs/en_US/Tutorial/QuickStart.rst

```diff
@@ -80,12 +76,20 @@ NNI is born to help the user do tuning jobs; the NNI working process is presente
    6:          Stop the experiment
    7: return hyperparameter value with best final result
 
-If you want to use NNI to automatically train your model and find the optimal hyper-parameters, you need to do three changes based on your code:
+.. note::
+
+   If you want to use NNI to automatically train your model and find the optimal hyper-parameters, there are two approaches:
+
+   1. Write a config file and start the experiment from the command line.
+   2. Config and launch the experiment directly from a Python file
+
+   In the this part, we will focus on the first approach. For the second approach, please refer to `this tutorial <HowToLaunchFromPython.rst>`__\ .
+
 
-Three steps to start an experiment
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 1: Define the Seach Space
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**Step 1**: Write a ``Search Space`` file in JSON, including the ``name`` and the ``distribution`` (discrete-valued or continuous-valued) of all the hyperparameters you need to search.
+Define a ``Search Space`` in the JSON file, including the ``name`` and the ``distribution`` (discrete-valued or continuous-valued) of all the hyperparameters you want to search.
```

Comment:
Maybe we can suggest embedded search space. It's more convenient in most cases.

Reply:
agree

Reply:
ok, so do we need to remove the JSON part? or suggest two ways to define the search space in the quickstart?

Reply:
I think in quick start we can remove JSON part.